src/pages/reports/index.astro

```diff
@@ -0,0 +1,39 @@
+---
+import { getCollection } from "astro:content";
+import Layout from "@/layouts/Layout.astro";
+import ContentItems from "@/components/ContentItems.astro";
+import PagesSection from "@/components/PagesSection.astro";
+import type { CollectionEntry } from "astro:content";
+import payloadFetch from "@/utilities/payload-fetch";
+import { formatDate } from "@/utilities/formatting";
+
+const collectionName = "reports";
+let reports: CollectionEntry<typeof collectionName>["data"][];
+
+if (!Astro.isPrerendered) {
+  const response = await payloadFetch(`${collectionName}?draft=true`);
+  const reportsData = await response.json();
+  reports = reportsData.docs;
+} else {
+  const reportsCollection = await getCollection(collectionName);
+  reports = reportsCollection.map((n) => n.data);
+}
+
+const items = reports.map((n) => ({
+  title: n.title,
+  date: formatDate(n.publishedAt),
```

Comment:
I think that reports should probably use Report Date for this field, not publishedAt